conda/meta.yaml

```diff
@@ -28,6 +28,7 @@ requirements:
         - tqdm
         - sqlalchemy
         - ipywidgets
+        - cftime>1.2.1
```

Comment:
I guess not a big deal to have this, but does seem irrelevant to the PR itself...?

Reply:
The tests failed because we had an old version of `cftime` installed. 

Arguably this should have been set in the PR where the tests were changed, but I thought we should insist on this otherwise tests fail.

Reply:
No problem!